app/controllers/requests/received_requests_controller.rb

```diff
@@ -1,4 +1,8 @@
 class Requests::ReceivedRequestsController < ApplicationController
   def index
+    receivers = Receiver.where(user: current_user)
+    @requests = []
+    receivers.each { |receiver| @requests << receiver.request }
+    @requests
```

Comment:
`has_many :received_requests, through: :receivers, source: :requests` is better in this case.
Cecile said so in the Le Wagon in this code.
https://github.com/hidehiro98/heytask/blob/master/app/models/user.rb

And I researched the difference between `class_name` and `source`.
https://stackoverflow.com/questions/13611265/rails-difference-between-source-and-class-name-in-models